packages/core-api/src/app/App.tsx

```diff
@@ -52,6 +52,7 @@ import {
 } from '../extensions/traversal';
 import { IconComponent, IconComponentMap, IconKey } from '../icons';
 import { BackstagePlugin } from '../plugin';
+import { pluginCollector } from '../plugin/collectors';
```

Comment:
Wherefore art thou, `../plugin/collectors`?

Reply:
ADR wants `*` index exports all the way to the root, so this is because we don't want to export `pluginCollector` from the package. Been considering changing that rule in the ADR though, it's a bit awkward, but entirely sure to what though.

Reply:
I commented thinking this was a new file missing from the PR, I see now it's already existing in source. Disregard.

Reply:
Yeah this change has been planned since a long time back :grin: